src/database/queries/getFromUsers.js

```diff
@@ -0,0 +1,11 @@
+const connection = require('../config/connection')
+ const getFromUsers = (userName) => {
+ 
+  return connection.query(`SELECT username FROM users WHERE username = '${userName}';`)
```

Comment:
By doing this you will be a subject to SQL injection attacks, you need to use parameterized queries 

[see here](https://node-postgres.com/features/queries)